ee/control/consumers/notificationconsumer/notify_consumer.go

```diff
@@ -128,7 +130,7 @@ func (nc *NotificationConsumer) notify(notificationToSend notification) {
 		return
 	}
 
-	if err := nc.runner.SendNotification(notificationToSend.Title, notificationToSend.Body); err != nil {
+	if err := nc.runner.SendNotification(notificationToSend.Title, notificationToSend.Body, notificationToSend.ActionUri); err != nil {
```

Comment:
I _might_ have found a bit of a race here. 

When I run launcher, it will hit this line before the desktop process has been spawned. Maybe send notification should return an error if there is no one to send a notification to? Or it should return the count of notifications sent?

Or maybe this isn't a big deal? Just don't want it eating notifications.

Reply:
Looks like I can have the desktop processes runner return an error if `r.uidProcs` is empty, which I think should do it! Thanks for pointing this out, I'll get it sorted

Reply:
Updated, and added a test for same as well 🙂